sites/all/modules/custom/bgimage/bgimage.module

```diff
@@ -122,9 +133,41 @@ function bgimage_menu() {
     'file' => 'bgimage.pages.inc',
   );
 
+  $items['images/raw/%/%/%'] = array(
```

Comment:
```suggestion
  $items['files/raw/%/%/%'] = array(
```

Reply:
Oh right, we're redirecting a url from legacy BugGuide, which uses 'images/...' instead of beta's 'files/...' (got confused trying to test with images created locally on new BugGuide).